Advanced_Projects/hardware-monitor/gui/dashboard.py

```
@@ -0,0 +1,51 @@
+                label = QLabel(label_text)
+                layout.addWidget(label)
+                tab.setLayout(layout)
+                return tab
+
+        app = QApplication(sys.argv)
+        window = DashboardWindow()
+        window.show()
+        sys.exit(app.exec_())
+    elif mode == 'web':
+        try:
+            import streamlit as st
```

Ruff (0.12.2)

`streamlit` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)